HackingCommon/nvapi.h

```diff
@@ -12632,3 +12632,5 @@ NVAPI_INTERFACE NvAPI_SYS_GetPhysicalGpuFromDisplayId(NvU32 displayId, NvPhysica
 #pragma pack(pop)
 
 #endif // _NVAPI_H
+
+void NvAPIOverride();
```

Comment:
nvapi.h is a third party header from nvidia and should not be altered. NvAPIOverride is our own big fat hack and should go in our headers.